en/99-apendice/2-repository-rules.md

```diff
@@ -0,0 +1,96 @@
+## Repository rules
+El principal material con el que trabajamos es código, con lo que se vuelve menester para la salud de todos los proyectos (y la nuestra) que mantengamos una serie de reglas para trabajar ordenadamente en los repositorios.
+
+### Gitflow
+Seguimos [gitflow](http://nvie.com/posts/a-successful-git-branching-model/) como modelo de branching y tagging, donde `master` actúa como branch de develop y mantenemos un branch `stable`.
+
+### Versionado
+Las versiones se notan como `MAJOR.MINOR.PATCH(-preX)`. Seguimos la política de [semantic versioning](http://semver.org/) para MAJOR, MINOR y PATCH.
+
+* MAJOR version when you make incompatible API changes,
+* MINOR version when you add functionality in a backwards-compatible manner, and
+* PATCH version when you make backwards-compatible bug fixes.
+
+Las `-pre1`, `-pre2`, etc se corresponden con los sucesivos deploys a staging hasta que la versión esté aprobada por QA.
+
```

Comment:
Every version is defined as `MAJOR.MINOR.PATCH(-preX)`. We follow [semantic versioning](http://semver.org/)'s conventions for MAJOR, MINOR and PATCH.

* MAJOR version when you make incompatible API changes,
* MINOR version when you add functionality in a backwards-compatible manner, and
* PATCH version when you make backwards-compatible bug fixes.

The `-pre1`, `-pre2`, etc are consecutive deploys to staging until such version is given the go ahead by the QA team.